offer-manager-domain/src/main/groovy/life/qbic/business/persons/affiliation/update/UpdateAffiliationOutput.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic.business.persons.affiliation.update
+
+import life.qbic.business.exceptions.DatabaseQueryException
+import life.qbic.business.persons.affiliation.Affiliation
+import life.qbic.business.persons.affiliation.AffiliationCategory
+
+/**
+ * The output interface for the Update Affiliation use case.
+ *
+ * @since 1.3.0
+ */
+interface UpdateAffiliationOutput {
+
+    /**
+     * Updates an affiliation entry
+     * @param oldAffiliation The affiliation to be updated
+     * @param newAffiliation The new affiliation entry
+     * @throws DatabaseQueryException
+     */
+    void updatedAffiliationCategory(Affiliation oldAffiliation, Affiliation newAffiliation) throws DatabaseQueryException
```

Comment:
Here it would suffice to return the changed affiliation with all the properties loaded from persistence. I do not see the benefit of passing through the unaltered entry (as it may no longer exist in that way). Maybe I am overlooking something?